src/box/lua/console.lua

```diff
@@ -646,6 +674,15 @@ end
 -- Print result to stdout
 --
 local function local_print(self, output)
+    if output:endswith(output_eos["lua"]) then
+        local new_local_eos
+        if (self.remote ~= nil) then
```

Comment:
Nit: please, don't do it in Lua (there is the same mistake in the first patch).

Reply:
My bad, it's reflexive. Fixed.